heat_infoblox/resources/grid_member.py

```diff
@@ -50,10 +50,13 @@ class GridMember(resource.Resource):
         DNS_DTC_HEALTH_SOURCE, DNS_DTC_HEALTH_SOURCE_ADDRESS,
         DNS_RPZ_QNAME_WAIT_RECURSE, DNS_USE_RPZ_QNAME_WAIT_RECURSE,
         DNS_LOG_DTC_GSLB, DNS_LOG_DTC_HEALTH, DNS_UNBOUND_LOGGING_LEVEL,
+        HA_PAIR, VIP, VIRTUAL_ROUTER_ID, LAN2_VIRTUAL_ROUTER_ID,
+        NODE1_HA_IP, NODE2_HA_IP, NODE2_LAN1_IP,
```

Comment:
I found that in "products/one/server/src/bin/clusterd/vrrp.c" defines next base MAC addresses:
```C
// Constants from RFC-2748 and RFC-5798
#define VRRP_MAC_ADDR_BASE   "00:00:5E:00:01:00"
#define VRRPv6_MAC_ADDR_BASE "00:00:5E:00:02:00"
```
I will use it to calculate MAC from VRID